README.md

```diff
@@ -40,6 +40,21 @@ header_name = {
 http_method = {
     type = "string",
     default = "GET"
+},
+add_token_claims_as_headers = {
+    type = "boolean",
+    default = true
+}
+frontier_header_prefix = {
+    type = "string",
+    default = "X-Frontier-"
+},
+appendable_claim_headers = {
+    type = "array",
+    default = BEARER_TOKEN_HEADERS,
```

Comment:
What does this default value mean? `BEARER_TOKEN_HEADERS`

Reply:
It should be list of `"sub", "org_ids"`?

Reply:
Also we don't need `appendable_claim_headers ` flag if we can use `add_token_claims_as_headers` as this check. That is we can make `add_token_claims_as_headers` type as array and if user has not passed any array item, it disables it?

Reply:
Merged both into one

Reply:
Readme still says `BEARER_TOKEN_HEADERS` there is no such var